src/status_im2/contexts/syncing/standard_authentication/enter_password.cljs

```diff
@@ -0,0 +1,97 @@
+                customization-color]}       (rf/sub [:profile/multiaccount])
+        {:keys [error processing password]} (rf/sub [:profile/login])
+        sign-in-enabled?                    (rf/sub [:sign-in-enabled?])
+        profile-picture                     (rf/sub [:profile/login-profiles-picture key-uid])
+        error                               (get-error-message error)]
+    [background/view true]
+    [:<>
+     [rn/view {:style style/enter-password-container}
+      [rn/view
+       [quo/text
+        {:accessibility-label :sync-code-generated
+         :weight              :bold
+         :size                :heading-1
+         :style               {:color         (colors/theme-colors colors/neutral-100 colors/white)
+                               :margin-bottom 4}}
+        (i18n/label :t/enter-password)]
+
+       [rn/view
+        {:flex-direction :row :margin-bottom 20 :margin-top 8}
+        [quo/context-tag {:blur? false} profile-picture display-name]]
+
+       [rn/view {:flex-direction :row :align-items :flex-end}
+        [rn/view {:flex 6}
+         [quo/input
+          {:type                :password
+           :blur?               true
+           :placeholder         (i18n/label :t/type-your-password)
+           :auto-focus          true
+           :show-cancel         false
+           :error?              (seq error)
+           :accessibility-label :password-input
+           :label               (i18n/label :t/profile-password)
+           :on-change-text      (fn [password]
+                                  (rf/dispatch [:set-in [:profile/login :password]
+                                                (security/mask-data password)])
+                                  (reset! entered-password password)
+                                  (rf/dispatch [:set-in [:profile/login :error] ""]))}]]]
+       (when (seq error)
+         [rn/view {:style style/error-message}
+          [quo/info-message
+           {:type :error
+            :size :default
+            :icon :i/info}
+           error]
+          [rn/touchable-opacity
+           {:hit-slop       {:top 6 :bottom 20 :left 0 :right 0}
+            :disabled       false
+            :active-opacity 1
+            :on-press       #(reset! entered-password %)}
+           [rn/text
+            {:style                 {:text-decoration-line :underline
+                                     :color                colors/danger-60}
+             :size                  :paragraph-2
+             :suppress-highlighting true}
+            (i18n/label :t/forgot-password)]]])
+
+       [rn/view
+        {:margin-top 45}
+        [quo/button
+         {:size                40
+          :type                :primary
+          :customization-color (or customization-color :primary)
+          :accessibility-label :login-button
+          :before              :i/reveal
+          :disabled            (or (not sign-in-enabled?) processing)
```

Comment:
There was some recent changes to button that got merged.

':before' -> ':icon-left'
':disabled' -> ':disabled?'
':style' -> ':container-style'

You can revert the changes you made to the button component as that would break all other uses in the app

Reply:
updated these props where I'm using this component.